idd/Energy+.idd.in

```diff
@@ -40777,6 +40786,7 @@ ZoneHVAC:EquipmentList,
        \key AirTerminal:SingleDuct:Uncontrolled
        \key ZoneHVAC:EnergyRecoveryVentilator
        \key ZoneHVAC:EvaporativeCoolerUnit
+       \key ZoneHVAC:ForcedAir:UserDefined
```

Comment:
Done.  Also added two missing equipment types based on [this case block](https://github.com/NREL/EnergyPlus/blob/d6868a1a5ea3b03cb2bc27ae97b4f8cbf76d66d0/src/EnergyPlus/DataZoneEquipment.cc#L574-L679)